packages/amazonq/src/api.ts

```diff
@@ -12,7 +13,11 @@ export default {
     chatApi: {
         async chat(request: GenerateAssistantResponseRequest): Promise<GenerateAssistantResponseCommandOutput> {
             const chatSession = new ChatSession()
-            return chatSession.chat(request)
+            return chatSession.chatSso(request)
+        },
+        async chatIam(request: SendMessageRequest): Promise<SendMessageCommandOutput> {
+            const chatSession = new ChatSession()
+            return chatSession.chatIam(request)
```

Comment:
this can be an optional parameter on `chat` instead of an entirely new function.